BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -106,17 +118,20 @@ def create  #:nodoc:
         @group.addlog_context(self,"Created by #{current_user.login}")
         flash[:notice] = 'Project was successfully created.'
         format.html { redirect_to :action => :index, :format => :html}
-        format.xml  { render :xml => @group, :status => :created, :location => @group }
+        format.xml  { render :xml  => @group.for_api, :status => :created }
+        format.json { render :json => @group.for_api, :status => :created }
       else
         @users = current_user.available_users.where( "users.login<>'admin'" ).order( :login )
```

Comment:
This line only necessary for HTML... also the check condition `login<>'admin'` is old stuff that really shoudl be rewritten using the proper methods for identifying the core admin user.